yeat/bandage/Snakefile

```diff
@@ -62,3 +62,29 @@ rule unicycler:
             filename = Path(gfa).stem
             shell("Bandage image {gfa} analysis/bandage/unicycler/{filename}.jpg")
         shell("touch {output[0]}")
+
+
+rule flye:
+    output:
+        "analysis/bandage/flye/.done"
+    input:
+        glob("analysis/flye/30-contigger/*.gfa"),
+        glob("analysis/flye/40-polishing/*.gfa"),
+        glob("analysis/flye/*.gfa")
+    run:
+        for gfa in input:
+            filename = Path(gfa).stem
+            shell("Bandage image {gfa} analysis/bandage/flye/{filename}.jpg")
+        shell("touch {output[0]}")
+
+
+rule canu:
+    output:
+        "analysis/bandage/canu/.done"
+    input:
+        glob("analysis/canu/unitigging/4-unitigger/*.gfa")
```

Comment:
Canu deprecated a lot of their "gfa" files since v2.1. The only gfa file they have left is "best edges" without repeating edges graph.

This is what skoren said:

> Unitigs + their gfa intentionally removed.

> They were just too fragmented (more fragmented than they needed to be) and unreliable (missing some edges) to be of useful. We expect in the future to add more reliable assembly graphs once we have finalized an improved contig strategy.

https://github.com/marbl/canu/issues/1786
https://github.com/marbl/canu/issues/1819